gematria/testing/python/basic_blocks_with_throughput.py

```diff
@@ -48,23 +51,35 @@
 
 def _get_basic_block_list_proto(get_annotated_blocks=False):
   """Loads basic blocks from test data."""
-  global _BASIC_BLOCKS
-  if _BASIC_BLOCKS is None:
-    runfiles_dir = os.environ.get('PYTHON_RUNFILES')
-    runfiles_env = runfiles.Create({'RUNFILES_DIR': runfiles_dir})
-    assert runfiles_env is not None
-    with open(
-        runfiles_env.Rlocation(
-            _ANNOTATED_BASIC_BLOCK_RESOURCE_PATH
-            if get_annotated_blocks
-            else _BASIC_BLOCK_RESOURCE_PATH
-        ),
-        'rt',
-    ) as f:
-      _BASIC_BLOCKS = text_format.Parse(
-          f.read(), throughput_pb2.BasicBlockWithThroughputListProto()
-      )
-  return _BASIC_BLOCKS
+  global _BASIC_BLOCKS, _ANNOTATED_BASIC_BLOCKS
+  if get_annotated_blocks:
+    if _ANNOTATED_BASIC_BLOCKS is None:
+      runfiles_dir = os.environ.get('PYTHON_RUNFILES')
+      runfiles_env = runfiles.Create({'RUNFILES_DIR': runfiles_dir})
+      assert runfiles_env is not None
+      with open(
+          runfiles_env.Rlocation(_ANNOTATED_BASIC_BLOCK_RESOURCE_PATH),
+          'rt',
+      ) as f:
+        _ANNOTATED_BASIC_BLOCKS = text_format.Parse(
+            f.read(), throughput_pb2.BasicBlockWithThroughputListProto()
+        )
+    assert _ANNOTATED_BASIC_BLOCKS is not None
+    return _ANNOTATED_BASIC_BLOCKS
+  else:  # get_annotated_blocks == False
+    if _BASIC_BLOCKS is None:
+      runfiles_dir = os.environ.get('PYTHON_RUNFILES')
+      runfiles_env = runfiles.Create({'RUNFILES_DIR': runfiles_dir})
+      assert runfiles_env is not None
```

Comment:
I'd move this out of the if/else statements (since it is the same for both cases).